rpm/Makefile

```diff
@@ -21,7 +21,7 @@ ARCH ?= `uname -m`
 
 build: source vendor vendor-rpms
 	rm -rf result
-	if [[ $(DISTRO) != "epel-7" ]]; then \
+	if [[ $(DISTRO) == "epel-6" || $(DISTRO) = amzn-* ]]; then \
```

Comment:
For consistency, could both sides use either `=` or `==`? Also (unless I'm missing something), the first half could also unquote `epel-6`.